src/classes/user.php

```diff
@@ -1,5 +1,5 @@
 <?php
-namespace Bitrix24\User;
```

Comment:
Can you move a class user into a folder? Without change a namespase becouse in other progects users can have an error if they load by full namespace.